### ISSUE TITLE
ear-utils: add possibility to replace chna

### ISSUE THREAD

**Issue by WernerBleisteiner** [ebu/ebu_adm_renderer#81]
All along, ear-utils offers the possibility to extract ADM's axml and chna chunks. Yet it only allows to add/replace (modified) axml (back). 
More and more there occur use cases that also require to modify chna data. So it'd be very helpful to do this with a similar process via ear-utils (other ways tend to be tricky and tedious)

**Comment by tomjnixon** [comment]
Yeah, i agree that would be useful. The issue I have is that it's not clear what file format should be used to specify the new CHNA items.

I suppose CSV would work, but it's not a pleasant format, and `dump_chna` already produces a format wich would be annoying to parse.

I think the `--gen-chna` option should cover most use cases, except those where you need more than one CHNA entry per track. This works by assuming that, for example, `ATU_00000002` is the second track.

**Comment by WernerBleisteiner** [comment]
Thanks. Yeah, that's an interesting question. 
Just found out (when tweaking chna with HEXFIEND) that ear-utils dumps an easier readable output.
Ideally, I'd prefer to take just that format, make changes and inject it again.
Is there a `--gen-chna` option in ear-utils? 
Mine does not seem to know it. 
Which version of EAR?

**Comment by tomjnixon** [comment]
>  Ideally, I'd prefer to take just that format, make changes and inject it again.

Yeah, that workflow makes sense. Probably the best approach would be to change the output of `ear-utils dump_chna` to something more sensible.

> Is there a --gen-chna option in ear-utils?
> Mine does not seem to know it.
> Which version of EAR?

It's an option for `ear-utils replace_axml` -- you should be able to see it with `ear-utils replace_axml --help`. It's been there since version 1!

**Comment by WernerBleisteiner** [comment]
Thanks @tomjnixon !
Gotta check it out, wasn't aware of that (some utilities seem to be too well hidden🙄)
